### PR TITLE
feat: update release info to v5.6.0

### PR DESCRIPTION
**Pull request by barmac**
image

Related to #3285

### CODE REVIEW

**Comment by philippfromme** [comment]
In the past, we avoided the name _linting_ and referred to it as _modeling guidance_ and other less technical terms. I was never a fan of this. It's similar to element templates which we call templates or connector templates because that's what they are. Furthermore, this is the Desktop Modeler, so its audience tends to be more technical. @christian-konrad what do you think?

**Comment by barmac** [comment]
Indeed we specifically change the release info against "lint" in https://github.com/camunda/camunda-modeler/commit/7f552dffe7499cbaa5e5f02a64266195de04de87

"Improved diagram problems integration" sounds a bit odd when we talk about this plugin template specifically: https://github.com/camunda/camunda-modeler-custom-linter-rules-plugin

**Comment by nikku** [comment]
You can also spin it the validation way.

> ### Improve Validation Infrastructure
> 
> Editor now supports errors and warnings as problem types. Contribute your own validators throw [linter plug-ins](...)

**Comment by barmac** [comment]
> You can also spin it the validation way.
> 
> > ### Improve Validation Infrastructure
> > Editor now supports errors and warnings as problem types. Contribute your own validators throw [linter plug-ins](...)

I'm going to use this. Thanks!

**Comment by barmac** [comment]
OK so this will point now to [the linter template project](https://github.com/camunda/camunda-modeler-custom-linter-rules-plugin).

image

**Comment by barmac** [comment]
I am merging this since there was a general 👍 from the team in the meeting.

**Comment by christian-konrad** [comment]
@barmac btw. regarding linting:

> We should not use this term now in public communications; yet **it is still valid to use the term in the context of customizing the Desktop Modeler, as keen users can use the bpmn-lint plugin to write their own linting rules**. We may offer custom linting rules in the Web Modeler in the future (e.g. in the context of governance rules), but until then, we don’t use this term.

From https://docs.google.com/document/d/1cWupck8oufHygtaueM6yFEHM9AG2EtKJeciairiskQs/edit#

So since we now allow this by default, we can absolutely talk about "adding own lint rules", or "own validation rules", both is fine I guess

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -44,8 +44,20 @@ export function ReleaseInfo(props) {
...
       <ul className="dashed">
         <li>
-          <h4>Forms preview</h4>
-          The form editor now supports a preview mode, which allows you to see how forms will look for the end user while they're being created.
+          <h4>Model all elements for Camunda 8</h4>
+          The BPMN editor now supports drawing all BPMN symbols while it also indicates which elements are not supported in the engine.
+        </li>
+        <li>
+          <h4>Improved validation infrastructure</h4>
+          Editor now supports errors and warnings as problem types. Contribute your own validators throw <a href="https://github.com/camunda/camunda-modeler-custom-linter-rules-plugin">linter plug-ins</a>.
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:52]: Should be `through` linter plug-ins?

Comment (barmac) [reply client/src/plugins/version-info/ReleaseInfo.js:52]: True, I missed that.

**Pull request merged**